lib/fizzy/execute.cpp

```diff
@@ -165,8 +175,49 @@ execution_result execute(Instance& instance, FuncIdx function, std::vector<uint6
             goto end;
         case Instr::nop:
             break;
+        case Instr::block:
+        {
+            const auto imm_begin = immediates;
+            const auto arity = read<uint8_t>(immediates);
+            const auto instr_diff = read<uint32_t>(immediates);
+            const auto imm_diff = read<uint32_t>(immediates);
+            label_context label{pc + instr_diff, imm_begin + imm_diff, arity, stack.size()};
+            labels.push_back(label);
```

Comment:
```suggestion
            labels.emplace_back(pc + instr_diff, imm_begin + imm_diff, arity, stack.size());
```

Reply:
I can change it to `emplace_back(label)` if that make a difference. But for debugging purposes it's useful to be able to inspect the create object before it is pushed.

Reply:
Well it should be `emplace_back(std::move(label))` then

Reply:
Maybe it looks more modern, but I hope you realize it makes no difference in this case.

Reply:
Ok true, it won't make any difference, we can leave it